src/order/V2DutchOrder.ts

```diff
@@ -327,11 +347,14 @@ export class V2DutchOrder extends V2Order {
   }
 
   /**
-   * @inheritdoc Order
+   *  recovers co-signer address from cosignature and full order hash
+   *  @param fullOrderHash The full order hash over (orderHash || cosignerData)
+   *  @param cosignature The cosignature to recover
+   *  @returns The address which co-signed the order
    */
   recoverCosigner(
     fullOrderHash: string,
-    cosignature: string = this.info.cosignature
+    cosignature: string = this.info.cosignature ?? "0x"
```

Comment:
any reason to add a default value here?

Reply:
I think because the cosignature is not there after the first stage of the auction